.gitignore

```diff
@@ -1,12 +1,12 @@
 .vs/
+.idea/
+.vscode/
 obj/
 bin/
-*.user
 artifacts/
-.vs/
-wwwroot
+wwwroot/
 node_modules/
-.pnpm-debug.log
-package-lock.json
 Localization/
-/.editorconfig
\ No newline at end of file
+*.user
+.pnpm-debug.log
+/.editorconfig
```

Comment:
I've restored `Localization/` and `/.editorconfig` here.

Reply:
Are they needed though? I assumed they are outdated, because I did a rebuild and nothing materialized.

Reply:
Well, I take your word for it. I'll remove them then.

Reply:
Are the other two occurrences of the same kind, i.e. obsolete?

Reply:
Yes.

Reply:
Reverted my commits.